tests/unit_tests/cpu/utils/grids/populators.cpp

```diff
@@ -56,7 +56,7 @@ GTEST_TEST(detray_grid, replace_populator) {
 
     // Create some bin data
     dvector<bins::single<dindex>> bin_data{};
-    bin_data.reserve(50);
+    bin_data.resize(50);
```

Comment:
What was going on in the past?

Reply:
The vector was not resized, leading to an out of bounds assertion when filling it by std::generate